index.md

```diff
@@ -7,7 +7,7 @@ redirect_from: /lang/en/
 
 ## Summary
 
-As an open-source maintainer, squash feature branches onto `master` and write
+As a software developer, i want to squash feature branches onto `master` and write
```

Comment:
grammar nit: `s/i/I`

Reply:
Also I'd mention that GH has the Squash and Merge function which I use heavily to enforce this myself while still lowering the barrier to entry for people. :)